lib/paypal_nvp.rb

```diff
@@ -58,6 +69,9 @@ def call_paypal(data)
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     end
 
+    http.open_timeout = @open_timeout
+    http.read_timeout = @read_timeout
```

Comment:
If they aren't set, they will be nil, so it will be timeout right away. Wasn't sure if we should set the default value. Maybe set both of them to 60s?